readme.md

```diff
@@ -50,6 +50,19 @@ Known protocols:
 A list of ISO-639-1 language codes describing the languages supported by the
 endpoint.
 
+### Timezone Information
+
+```js
+{
+    "timezone": "Europe/Berlin"
+}
+```
+
+For endpoints that assume times in requests to be in a specific local timezone, the `timezone` field
+should contain the [IANA timezone identifier](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) of the
+expected timezone. This timezone can also be applied by clients to times in results returned by the endpoint,
+particularly when the results are lacking explicit timezone information.
+
```

Comment:
+1 for a single timezone specifier covering both requests and responses. 

Personally, I'd explicitly state that we expect responses to either contain timezone information (→ the "timezone" field is only relevant for requests and may be used as a default timezone for response representation purposes) or, lacking that, to be presented in "timezone". We don't expect an API to return data which claims to be timezone X (≠ "timezone" value) but actually is timezone Y (as specified in "timezone").

I may be overcatious here though, so feel free to leave it as-is if you think it's sufficiently clear.

Reply:
> We don't expect an API to return data which claims to be timezone X (≠ "timezone" value) but actually is timezone Y (as specified in "timezone").

Why not? I'd highly appreciate it if, for a train starting in Paris, the SNCF API endpoint (with is main operating area being `Europe/Paris`) to return arrivals times  in/with `Europe/London`, because then clients would be able to

- show the arrival time in London's *local wall clock time*, and
- indicate that the arrival is in a different timezone (+ offset) than the departure.

Reply:
Absolutely! I meant that we do not expect a case such as the following:

* The SNCF API endpoint returns "Thu, 14 Jan 2021 22:20:47 GMT"
* however, our `timezone` field specifies Europe/Paris, so we assume that the SNCF API timezone is bogus and it's actually "Thu, 14 Jan 2021 22:20:47 CET" (and, thus, 21.20 GMT)

I.e., "applying the timezone to results" does not mean "overwriting timezones in results".